buildSrc/version.properties

```diff
@@ -41,6 +41,7 @@ commonscodec      = 1.15
 
 # plugin dependencies
 aws               = 1.12.270
+aws_v2            = 2.20.12
```

Comment:
This one is discussed in https://github.com/opensearch-project/OpenSearch/issues/7312, we'd like AWS SDK to be upgraded rather than running v1 and v2 side-by-side.

Reply:
Crypto plugin doesn't exist today and hence there's no usage of v1 SDK in crypto. It will require v2 SDK though to build on top of v2 encryption SDK and hence you see a version property added here. Migration of v1 to v2 is independent of crypto plugin and will be taken independently as part of repository s3 plugin.

Reply:
It's not about usage. We discussed in https://github.com/opensearch-project/OpenSearch/pull/7216 having both libraries side-by-side irrespective of whether we're mixing v1 code or not mixing v2 code. It's the same codebase and we wouldn't include multiple versions of a library in it with one component building with v1 and another with v2.

Reply:
Got your point. This property addition would mean that now in single bundle we have both SDKs. 
Just thinking that how we should handle this case then properly. There might be many plugins both within OpenSearch project and outside OpenSearch which would be using v1 and v2 SDKs. Making the change here to remove v1 and keep v2 would mean that all of these plugins should be upgraded in one shot. And if we plan to do it incrementally, then we will beat our principle of keeping single SDK. 
What should be our next course of action in this case then?

Reply:
AFAIK OpenSearch core is the only repo that has any dependency on AWS SDK for the discovery and repository plugins. My preference would be to do #7312. I started in https://github.com/dblock/OpenSearch/tree/aws-sdk-v2, but finding little time to focus on it, so any help appreciated.